docs/decisions/0004-release-captain.md

```diff
@@ -0,0 +1,32 @@
+# Context and Problem Statement
+
+We want to share the responsibility of release management. Each week of the
+release, one team member will be responsible for all release-related tasks.
+This team member will be the release captain.
+
+This document defines and agrees on the responsibilities of said release captain.
+
+## Considered Options
+
+### Steps to follow on a release
+
+- Tag s3gw-ui repository main branch with version: vX.Y.Z
```

Comment:
there should be 5 tag indications in the document, 4 to do before the final release note step:

- ceph
- s3gw-tools
- s3gw-ui
- s3gw-charts

1 final tag and contextual release notes:

- s3gw

Reply:
I'm not entirely sure we're tagging those repos anymore. I believe we now only tag the `s3gw` repo

Reply:
In fact the text should probably read `Tag the s3gw repository main branch with version: vX.Y.Z`